source/crud/tests/v2/bulkWrite-arrayFilters.yml

```diff
@@ -65,3 +65,60 @@ tests:
         data:
           - {_id: 1, y: [{b: 2}, {b: 2}]}
           - {_id: 2, y: [{b: 0}, {b: 2}]}
+  -
+    description: "BulkWrite with arrayFilters using update pipelines"
+    operations:
+      -
+        name: "bulkWrite"
```

Comment:
The final PR should also add new tests files (with `minServerVersion`) for the `updateOne`, `updateMany`, and `findOneAndUpdate` CRUD methods. Those can be created once we're all set with this `bulkWrite` test, since I expect it will just entail some copying and slight modifications to APM.

Reply:
The new test file has test cases for these operations.